test/integration/sagemaker/test_mnist.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+from sagemaker.tensorflow import TensorFlow
+
+
+@pytest.mark.skip_gpu
+def test_mnist_cpu(sagemaker_session, ecr_image, instance_type):
+    instance_type = instance_type or 'ml.c4.xlarge'
```

Comment:
is it possible to have this defaulting behavior in the fixture definition?

Reply:
yeah. I can move it there.